iocore/net/I_UDPPacket.h

```diff
@@ -42,13 +43,47 @@
 class UDPPacket
 {
 public:
-  virtual ~UDPPacket() {}
-  virtual void free(); // fast deallocate
-  void setContinuation(Continuation *c);
-  void setConnection(UDPConnection *c);
-  UDPConnection *getConnection();
-  IOBufferBlock *getIOBlockChain();
-  int64_t getPktLength() const;
+  UDPPacket();
+  ~UDPPacket();
+  void free(); // fast deallocate
+
+  void
+  setContinuation(Continuation *c)
+  {
+    cont = c;
+  }
+
+  void
+  setConnection(UDPConnection *c)
```

Comment:
I'd rather have function definitions out of class definition for readability especially if functions have multiple lines, although having those in a header file is fine if those don't bring additional dependency.

Reply:
I put these back in the header to try to restore some inline-ability.  I can move them outside of the class though if you want.

Reply:
> I put these back in the header to try to restore some inline-ability.

Sure, that's fine. We do it at other places (e.g. [ProxyTransaction.h](https://github.com/apache/trafficserver/blob/master/proxy/ProxyTransaction.h))